cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4658,4 +4658,16 @@
       system log start 1 day ago to collect logs
     </description>
   </property>
+
+  <property>
+    <name>dataproc.cconf.idleTTL.minutes</name>
```

Comment:
Major: Let's not add in cdap-default. Have it a constant in the code. When the key-val property map doesn't have idleTTL set, use the hard-coded default value, otherwise use whatever in the passed in key-value property map